### PR TITLE
Added default values

### PR DESCRIPTION
**Pull request by theofficetroll**

### CODE REVIEW

**Comment by theofficetroll** [comment]
https://trello.com/c/7luSjKTn

File: src/components/item.jsx [hunk 1]
```diff
@@ -21,15 +24,38 @@ class Item extends React.Component {
...
+  }
+
+  numberGenerator (value, max) {
+    var hash = 0;
+    for (var i = 0; i < value.length; i++) {
+      hash = (hash << 5) + hash + value.charCodeAt(i);
+      hash = hash & hash; // Convert to 32bit integer
+      hash = Math.abs(hash);
+    }
+    return hash % max;
```
Comment (redperezz) [inline src/components/item.jsx:58]: ✓ Great defaults!

File: src/components/tier.jsx [hunk 2]
```diff
@@ -46,18 +55,44 @@ class Tier extends React.Component {
...
+      hash = (hash << 5) + hash + value.charCodeAt(i);
+      hash = hash & hash; // Convert to 32bit integer
+      hash = Math.abs(hash);
+    }
+    return hash % max;
+  }
+
+
+  render() {
+    let costText;
```
Comment (redperezz) [inline src/components/tier.jsx:85]: ✓ very good!

**Review by redperezz — APPROVED** [review]
Very nice job on your defaults! I'm working hard to get the Item service up and running and should be able to do it by today or tomorrow! Again great work!

**Pull request merged**